sys/backend_proxy.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Template = require('hyperswitch').Template;
+
+module.exports = (options) => {
+    const backendURITemplate = new Template({
+        uri: `${options.backend_host_template}/{{path}}`
+    });
+    return {
+        spec: {
+            paths: {
+                '/{+path}': {
+                    all: {
+                        operationId: 'proxy'
+                    }
+                }
+            }
+        },
+        operations: {
+            proxy: (hyper, req) => {
+                // Add the proxied module name to the path.
```

Comment:
Could you add a slightly-longer desc and an example for easier comprehension, please?